lib/tools/misc/bezierTools.js

```diff
@@ -0,0 +1,182 @@
+define([
+    './arrayTools'
+],
+function(
+    arrayTools
+){
+    "use strict";
+    var epsilon = 1e-12;
+
```

Comment:
to make it better visible what our dependencies are:
```
    var epsilon = 1e-12
      , calcBounds = arrayTools.calcBounds
     ;
```

and then later use just `calcBounds`